integtest/src/main/scala/com/pingcap/spark/DAGTestCase.scala

```diff
@@ -145,9 +140,20 @@ class DAGTestCase(prop: Properties) extends TestCase(prop) {
     s"select tp_bigint%1000 a, count(*) from $TABLE_NAME group by (tp_bigint%1000) having sum(tp_bigint%1000) < 100 order by a"
   )
 
-  def createArithmeticAgg(): List[String] = colSet.map((col: String) => s"select sum($col),avg($col),min($col),max($col) from $TABLE_NAME group by $col ${orderBy(col)}").toList
+  def createSimpleSelect(): List[String] = colSet.map((col: String) => select(col) + orderBy(col) + limit()).toList
+
+  def createArithmeticAgg(): List[String] = colSet.flatMap((col: String) =>
+    Seq(s"select sum($col) from $TABLE_NAME",
```

Comment:
You may just use List() instead of Seq()

Reply:
Why?

Reply:
This is a flatMap, even using List(), result has to call `.toList`

Reply:
my bad, I forgot colSet is a mutable set.